debezium-connector-mysql/src/test/resources/ddl/geometry_test.sql

```diff
@@ -16,9 +16,10 @@ CREATE TABLE dbz_222_point (
   expected_srid INT,
   PRIMARY KEY (id)
 ) DEFAULT CHARSET=utf8;
-INSERT INTO dbz_222_point VALUES (default,GeomFromText('POINT(1 1)'), 1.0, 1.0, NULL);
-INSERT INTO dbz_222_point VALUES (default,GeomFromText('POINT(8.25554554 3.22124447)', 4326), 8.25554554, 3.22124447, 4326);
-INSERT INTO dbz_222_point VALUES (default,GeomFromText('POINT(0 0)', 1234), 0.0, 0.0, 1234);
+INSERT INTO dbz_222_point VALUES (default,ST_GeomFromText('POINT(1 1)'), 1.0, 1.0, NULL);
+INSERT INTO dbz_222_point VALUES (default,ST_GeomFromText('POINT(8.25554554 3.22124447)', 4326), 8.25554554, 3.22124447, 4326);
+-- Invalid SRID is not supported in MySQL 8
```

Comment:
Maybe this should be extracted into a separate file which isn't used with MySQL 8?